var/ramble/repos/builtin/package_managers/spack-lightweight/package_manager.py

```diff
@@ -1128,11 +1128,17 @@ def get_package_path(self, package_spec):
             name_match = name_regex.match(name)
             if name_match:
                 name = name_match.group("name")
+        else:
+            # Remove newlines and whitespace from name
+            name = name.replace("\n", "").strip()
```

Comment:
Just curious why spack would return something with newlines?

Reply:
Maybe just so it's printed to the screen nicely?

i.e.

```
$ spack -D software/wrfv4-intel-mpi/ find --format={name} wrf
wrf
$ 
```

Reply:
Ah right